### PR TITLE
Clean up translated property observers on destroy.

### PR DESCRIPTION
**Pull request by nullrocket**
#188

I couldn't find an event that matched .on('destroy') at http://emberjs.com/api/classes/Ember.View.html  both **willDestroyElement** and **willClearRender** are needed though.

### CODE REVIEW

**Comment by jamesarosen** [comment]
Nice work. I fixed the test, did a little whitespace cleanup, and committed as 524837cb711c3c204d7a12aff3241f48a9616e5f.

File: lib/i18n.js [hunk 1]
```diff
@@ -109,17 +109,27 @@
     },
 
     TranslateableProperties: Ember.Mixin.create({
+        _translationObserver :  function(sender, key){
```
Comment (jamesarosen) [inline lib/i18n.js]: This `key` already has the `'Translation'` suffix since that's the property we're observing.

**Comment by nullrocket** [comment]
It was late I gave up for the night after travis failed. I appreciate the fix to the PR, I was in a hurry and just edited in Github since my local repo has too many changes and would have caused a huge pain to do a pull from :)

**Comment by jamesarosen** [comment]
I accidentally left a typo in when I pushed. I have a fix for the new JSHint error ready as soon as I get to work.

Thanks for digging in and finding this!

**Pull request closed without merging**